src-graphql/collection-default-fields.js

```diff
@@ -0,0 +1,3 @@
+import imageQuery from './image-query';
+
+export default ['id', 'handle', 'updatedAt', 'title', ['image', imageQuery()]];
```

Comment:
Unless this is being tested separately, I don't think it needs to be its own module.